### ISSUE TITLE
Separate different submisstion by Date Submitted

### ISSUE THREAD

**Issue by spiderPan** [olestourko/open-tech-pay#9]
Submission could be grouped by submitted date, so that same dev will not be count two times if he submitted in different years.

**Comment by olestourko** [comment]
Makes sense, I've been meaning to do this for a while and add a per-year view. I guess its time to actually implement this. Thanks Pan!
